src/python/WMCore/WMRuntime/Bootstrap.py

```diff
@@ -231,12 +237,18 @@ def createInitialReport(job, reportName):
     # $MACHINEFEATURES/total_cpu: number of configured job slots
     # $JOBFEATURES/hs06_job: HS06 score available to your job
     # $JOBFEATURES/allocated_cpu: number of allocated slots (=8 in case of a multicore job
+
+    machineFeaturesFile = os.environ.get('MACHINEFEATURES/hs06')
```

Comment:
the env var is just `MACHINEFEATURES`

Reply:
That is why I get empty dict. :-) Thanks